### DISCUSSION TITLE
Swagger Response With RaiseException

### DISCUSSION THREAD

**Discussion by TiewKH** [fastapi/fastapi#7676] [Questions] [answered]
### First check

* [X] I used the GitHub search to find a similar issue and didn't find it.
* [X] I searched the FastAPI documentation, with the integrated search.
* [X] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

In the auto-generated Swagger UI, is there any way to include the RaiseException responses?

### Additional context

For example, if I follow the [guide here](https://fastapi.tiangolo.com/tutorial/handling-errors/), to raise a HTTPException 404 error when the record is not found in the database, I want it to appear in the Swagger UI.

**Comment by phy25** [comment, marked as answer]
https://fastapi.tiangolo.com/advanced/additional-responses/#additional-response-with-model

**Comment by TiewKH** [comment]
> https://fastapi.tiangolo.com/advanced/additional-responses/#additional-response-with-model

Thanks for the link.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :bowing_man: :cake: 

And thanks @TiewKH for reporting back and closing the issue. :+1:
